packages/mgt-react/scripts/generate.js

```diff
@@ -24,7 +24,7 @@ const gaTags = new Set([
   'search-results',
```

Comment:
Shouldn't the tags here match the registered components exports?

Reply:
yes, this list and that set of functions called in registerMgtComponents should match.